<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sequencing Manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="allTaskTextLabel">
       <property name="minimumSize">
        <size>
         <width>96</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>All Task</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="allTaskProgressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="currentTaskTextLabel">
       <property name="minimumSize">
        <size>
         <width>96</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Current Task</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="currentTaskProgressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="allTaskInputTextLabel">
       <property name="minimumSize">
        <size>
         <width>96</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>State</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="seqStateLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="currentTaskName">
       <property name="minimumSize">
        <size>
         <width>360</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Task</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="outputFolderButton">
       <property name="text">
        <string>Choose</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="outputfolder">
       <property name="minimumSize">
        <size>
         <width>360</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>output folder</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="startSeqButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pauseButton">
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resumeButton">
       <property name="text">
        <string>Resume</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="abortButton">
       <property name="text">
        <string>Abort</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
